MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -2,11 +2,197 @@ import UIKit
 
 final class MovieQuizViewController: UIViewController {
     // MARK: - Lifecycle
+    
+    @IBOutlet weak private var counterLabel: UILabel!
+    @IBOutlet weak private var imageView: UIImageView!
+    @IBOutlet weak private var textLabel: UILabel!
+    
+
+
+    private var currentQuestionIndex: Int = 0
+    private lazy var currentQuestion  = questions[currentQuestionIndex]
```

Comment:
Не больше одно пробела, если это не табуляция
```suggestion
    private lazy var currentQuestion = questions[currentQuestionIndex]
```